e2e/tests/components/date-time-picker/date-time-picker.e2e-spec.ts

```diff
@@ -200,4 +200,13 @@ describe('Date Time Picker Tests', () => {
         }
 
     });
+
+    it('should not display horizontal line when select adjacent month', async () => {
```

Comment:
Here again the reference to the horizontal line should be removed, because it is theme specific.